code/__DEFINES/vendors.dm

```diff
@@ -21,10 +21,11 @@
 #define MARINE_CAN_BUY_COMBAT_ARMOR (1<<19)
 #define MARINE_CAN_BUY_COMBAT_R_POUCH (1<<20)
 #define MARINE_CAN_BUY_COMBAT_L_POUCH (1<<21)
+#define MARINE_CAN_BUY_FIELD_KIT (1<<22)
 
-#define MARINE_CAN_BUY_KIT (1<<22)
+#define MARINE_CAN_BUY_KIT (1<<23)
 
-#define MARINE_CAN_BUY_ALL ((1<<23) - 1)
+#define MARINE_CAN_BUY_ALL ((1<<24) - 1)
```

Comment:
this 100% seems like it should break something somewhere

Reply:
it actually doesn't. 24) - 1 is the only way it will work.